{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "d2c12da7_37146f82",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 192
      },
      "writtenOn": "2022-06-01T10:20:20Z",
      "side": 1,
      "message": "i think we should go with this\n\nsee analysis in https://gist.github.com/marios/9553f5e8a826bd13e2eb35f2addb016c for first 2x runs of this patch\n\nthe 3rd run also fails on tempest\n\n\none test that fails across all is test_delete_server_while_in_suspended_state but slightly differently (2x failed to reach ACTIVE status https://bugs.launchpad.net/tripleo/+bug/1964940 and 1x Unexpected response code received)",
      "revId": "e2f191cd5d0c49be99a96b7bdcd52edef54e9142",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    }
  ]
}